data-raw/data_mile1.R

```diff
@@ -15,6 +15,10 @@
 ###############################################################################
 # Helper functions and load packages
 ###############################################################################
+cv_2_sd <- function(x) {
+  sqrt(log(x^2 + 1))
+}
+
```

Comment:
Now just the sd calculated from the CV rather than the log of the standard deviation. Thanks again @Bai-Li-NOAA for catching this and @Andrea-Havron-NOAA for confirming what we need.

Reply:
looks good!